resource-managers/mesos/src/test/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterSchedulerSuite.scala

```diff
@@ -30,6 +30,7 @@ import org.mockito.Mockito._
 import org.scalatestplus.mockito.MockitoSugar
 
 import org.apache.spark.{LocalSparkContext, SparkConf, SparkFunSuite}
+import org.apache.spark.internal.config._
```

Comment:
Hi, @farhan5900 . Could you run `dev/lint-scala`? This should be moved to at line 37.

Reply:
Fixed it.